### PR TITLE
Added Configurations

### PR DESCRIPTION
**Pull request by KokhanO**

### CODE REVIEW

**Review by Crowy-o — COMMENTED** [review]
Besides comment - okey for now

File: utils/include/CHDDInfoSettings.h [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+	[[nodiscard]] std::string GetFileName() const;
+	[[nodiscard]] bool GetCheckHdd() const;
+	[[nodiscard]] int GetCountType () const;
+
+private:
+	std::shared_ptr<CDataReader> p_data_reader_;
+	std::string file_name_;
+	bool check_hdd_;
+	int count_type_;
```
[the remark this answers is not in the document]
Comment (KokhanO) [reply utils/include/CHDDInfoSettings.h]: fixed

File: utils/include/CDataReader.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+bool CDataReader::TryToConvertToBool(const std::string& data_to_convert, bool& return_data)
+{
+	const std::string on = "enable";
+	const std::string off = "disable";
+	if (on == data_to_convert)
+	{
+		return_data = true;
+		return true;
+	}
```
[the remark this answers is not in the document]
Comment (KokhanO) [reply utils/include/CDataReader.cpp:13]: Discussed

File: utils/include/CComunicationSettings.h [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	void ReadConfigurationFromFile();
+
+	[[nodiscard]] bool GetBlocking() const;
+	[[nodiscard]] int GetSocketTimeout() const;
+	
+private:
+	std::shared_ptr<CDataReader> p_data_reader_ = nullptr;
+
+	bool blocking_ = false;
+	int socket_timeout_ = 5;
```
[the remark this answers is not in the document]
Comment (KokhanO) [reply utils/include/CComunicationSettings.h]: fixed

**Pull request merged**